docs/howto/run_material_mapping.rst

```diff
@@ -87,6 +87,58 @@ Then edit the config-map.json file
 
    $ python3 <source>/Examples/Scripts/MaterialMapping/configureMap.py geometry-map.json config-map.json
 
+Geometry visualisation and preparation of JSON files
```

Comment:
Last time while writing the doc I was told that ACTS moved away from rst to md. Not sure it applies here.